File: 02_ProgrammingFundamentals/_04_Methods/_01_Lecture/_03_PrintingTriangle.java
package _02_ProgrammingFundamentals._04_Methods._01_Lecture;

import java.util.Scanner;

public class _03_PrintingTriangle {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Create method to print a triangle
        printTriangle(n);

        scan.close();
    }

    private static void printTriangle(int n) {
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.print(j + " ");
            }

            System.out.println();
        }

        for (int i = n - 1; i > 0; i--) {
            for (int j = 1; j <= i; j++) {
                System.out.print(j + " ");
            }

            System.out.println();
        }
    }
}

File: 24_C#Fundamentals/_04_Methods/_01_Lecture/_03_PrintingTriangle/_03_PrintingTriangle.cs
using System;

public class _03_PrintingTriangle
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Create method to print a triangle
        printTriangle(n);
    }

    private static void printTriangle(int n)
    {
        for (int i = 1; i <= n; i++)
        {
            for (int j = 1; j <= i; j++)
            {
                Console.Write(j + " ");
            }

            Console.WriteLine();
        }

        for (int i = n - 1; i > 0; i--)
        {
            for (int j = 1; j <= i; j++)
            {
                Console.Write(j + " ");
            }

            Console.WriteLine();
        }
    }
}
